ice/mvc/model.zep

```diff
@@ -147,14 +151,16 @@ abstract class Model extends Arr implements \Serializable
      */
     public function load(var filters, array options = [])
     {
-        var result, instances, data;
+        var result, instances, data, model;
 
         let result = this->db->find(this->from, filters, options),
             instances = [];
 
         if result->count() {
             for data in iterator(result) {
-                let instances[] = create_instance_params(get_called_class(), [null, data]);
```

Comment:
As I remember I had some issues with static implementation on zephir and that's why I used
```zep
create_instance_params(get_called_class(), [null, data]);
```